### DISCUSSION TITLE
Could you please give me some guidance

### DISCUSSION THREAD

**Discussion by zjgzh001** [azerothcore/azerothcore-wotlk#16230] [General]
The newly downloaded azbotcore compilation has no errors. When starting Worldserver after installation, the following error occurred:
Incorrect format (invalid format string) at '
E: azbotcoreserver azerothcore wotlk src server game Globals Object Manager. cpp: 2888'
The database does not have ENtry 80070, etc
Hello! Can you help repair it?
![QQ截图20230511151756](https://github.com/azerothcore/azerothcore-wotlk/assets/133203282/5589b38c-a3f8-4677-b851-d0534c4311a7)

**Comment by Kitzunu** [comment]
Well first of all, it is impossible for a clean install to generate any errors as all CI's are passing.

So in your case you are either using an outdated Database, or outdated core.

**Reply by zjgzh001** [reply]
First of all, thank you for your response. Secondly, I would like to say that this is indeed the latest Azerothcore wotlk from GIT. After installation, there was an error E: azbotcoreserever Azerothcore wotlk src server game Globals Object Manager. cpp: 2888 '. I have reinstalled the Windows system and then reinstalled Azerothcore wotlk, but the error still exists. I don't know what happened now, please provide more guidance

**Reply by Kitzunu** [reply]
Either your DB is outdated or the source is. That is the only reason for the error.

Make sure you pull head and not origin.
Run CMake
Rebuild

**Reply by zjgzh001** [reply]
Thank you very much for your reply! After updating the core, it seems that the problem has been fixed!
